notes/WT/week3.html

```diff
@@ -150,9 +150,13 @@
 
 <p><strong>How it works</strong></p>
 
-<p>Narrowband data is transformed into broadband spread spectrum signals with the same total energy ∴ original data is <strong>spread</strong> over a broader band with lower power. The low-power signals appear as background noise to unintended receivers ∴ narrowing frequency band interference.</p>
+<p>Narrowband data is transformed into broadband spread spectrum signals with the same total energy ∴ original data is <strong>spread</strong> over a broader band with lower power. The low-power signals appear as background noise to unintended receivers ∴ resolving narrow frequency band interference.</p>
 
 <p><em>It’s ok I’m confused too</em></p>
+<!-- What I think:
+The power from the narrowband is spread across the broadband as separate signals and since there is a bigger area of spread, the power of each signal is significantly low, allowing unintended receivers to ignore it as background noise.
+What do you think? See this image: http://mwrf.com/site-files/mwrf.com/files/uploads/2014/11/22Q_F1.gif-->
```

Comment:
Yep picture makes a lot of sense. I'll try to include it in explanation and add throw the pic in as well. Leave this comment here for now